api/__tests__/helpers/urls.test.js

```diff
@@ -0,0 +1,25 @@
+const cleanUrlsAndEmails = require("../../helpers/urls").cleanUrlsAndEmails;
```

Comment:
I noticed the code editor did that, and assumed it was the es lint requirements we'd set a while ago. I'll change them, we may want to update the eslint.

Reply:
We don't actually have any ES Lint or Prettier settings set up on the API, only the front-end app.  It would be a great one to do (and probably wouldn't take long) -- the ticket's here if you fancy it https://sta2020.atlassian.net/browse/SVA-293